js/main.js

```diff
@@ -0,0 +1,62 @@
+
+var $addCard = document.querySelector('#add-card');
+var $searchBarDiv = document.querySelector('#search-bar');
+var $cardCount = document.querySelector('#card-deck-count');
+var $infoText = document.querySelector('#click-to-add');
+var $noDeck = document.querySelector('#no-deck');
+var $newDeck = document.querySelector('.new-deck');
+var $search = document.querySelector('#search');
+var $myDeck = document.querySelector('#my-deck');
+var $myDeckView = document.querySelector('#my-deck-view');
+var $logo = document.querySelector('.logo');
+var $searchBar = document.querySelector('.search-bar');
+
+function searchBarAppears(event) {
+  event.preventDefault();
+  $newDeck.className = 'new-deck hidden';
+  $cardCount.className = 'minor-padding column-full row space-between hidden';
+  $searchBarDiv.className = 'colun-full row justify-center search-bar-padding';
+  $infoText.className = 'row justify-center align-center hidden';
+  $myDeckView.className = 'row justify-center align-center';
+}
+
+function myDeckAppears(event) {
+  event.preventDefault();
+  $newDeck.className = 'new-deck';
+  $cardCount.className = 'minor-padding column-full row space-between hidden';
+  $searchBarDiv.className = 'colun-full row justify-center hidden';
+  $infoText.className = 'row justify-center align-center hidden';
+  $myDeckView.className = 'row justify-center align-center';
+}
+
+function newDeck(event) {
+  event.preventDefault();
+  $newDeck.className = 'new-deck hidden';
+  $cardCount.className = 'minor-padding column-full row space-between';
+  $searchBarDiv.className = 'colun-full row justify-center hidden';
+  $infoText.className = 'row justify-center align-center';
+  $myDeckView.className = 'row justify-center align-center';
+}
```

Comment:
This looks like something that might be suited to making a single `switchView` function like you did in `code-journal`. If more views are necessary in the future, you don't want to need to make a new function each time.

Reply:
I recommend reviewing `javascript-view-swapping` for a reminder on how to effectively make a single function to set the current view.